tests/CMakeLists.txt

```diff
@@ -27,7 +27,11 @@ function(parsec_addtest_executable lang target)
   set(options NODEFAULTBUILD)
   set(multiValueArgs SOURCES)
   cmake_parse_arguments(PARSEC_ADDTEST "${options}" "" "${multiValueArgs}" ${ARGN})
-  add_executable(${target} ${PARSEC_ADDTEST_SOURCES})
+  if("${CMAKE_SOURCE_DIR}" STREQUAL "${PROJECT_SOURCE_DIR}")
```

Comment:
Should we document somewhere that tests are only built in stand-alone builds?

Reply:
There is a note in the comments of this macro